Makefile

```diff
@@ -28,6 +28,7 @@ vimdoc: build/bin/vimdoc
 
 configreference:
 	./bin/phpactor development:configuration-reference > doc/reference/configuration.rst
+	./bin/phpactor development:command-reference > doc/reference/rpc_command.rst
```

Comment:
if done as i suggested this would be `development:document rpc-commands` and `development:document configuration`, if that makes sense